pandas/_libs/tslibs/nattype.pyx

```diff
@@ -95,10 +95,6 @@ cdef class _NaT(datetime):
     # higher than np.ndarray and np.matrix
     __array_priority__ = 100
 
-    def __hash__(_NaT self):
-        # py3k needs this defined here
-        return hash(self.value)
```

Comment:
are you sure this can be removed? (do you know why?)

Reply:
There are two `__hash__` definitions for this class so I suppose the second one takes priority

Reply:
Ah, that's a good reason to remove it ;)